appveyor.yml

```diff
@@ -76,6 +76,7 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: conda install --yes --quiet conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
+    - cmd: conda install --yes --quiet conda-build=1.20.0
```

Comment:
If this is proven we should not merge this, but fix it in [`conda-forge-build-setup`](https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/master/recipe/run_conda_forge_build_setup_win#L12) instead.